contrib/linux/dosbox-staging.metainfo.xml

```diff
@@ -9,16 +8,24 @@
   </provides>
   <content_rating type="oars-1.1" />
   <name>DOSBox Staging</name>
-  <summary>DOSBox Staging is modernized DOSBox soft-fork</summary>
+  <summary>DOS/x86 emulator focusing on ease of use</summary>
   <description>
     <p>
-      DOSBox Staging is a DOS-emulator using SDL 2 for easy portability to different platforms.
+      DOSBox Staging is a full x86 CPU emulator (independent of host architecture),
+      capable of running DOS programs that require real or protected mode.
     </p>
     <p>
-      DOSBox emulates a 286/386 realmode CPU, Directory FileSystem/XMS/EMS, a SoundBlaster card
-      for excellent sound compatibility with older games... You can "re-live" the good old days
-      with the help of DOSBox Staging, it can run plenty of the old classics that don't run on your
-      new computer!
+      It features built-in DOS-like shell terminal, emulation of several PC variants
+      (IBM PC, IBM PCjr, Tandy 1000), CPUs (286, 386, 486, Pentium I), graphics
+      chipsets (Hercules, CGA, EGA, VGA, SVGA), audio solutions (Sound Blaster,
+      Gravis UltraSound, Disney Sound Source, Tandy Sound System), CD Digital Audio
+      emulation (also with audio encoded as FLAC, Opus, OGG/Vorbis, MP3 or WAV),
+      joystick emulation (supports modern game controllers), serial port emulation,
+      IPX over UDP, GLSL shaders, and more.
+    </p>
+    <p>
+      DOSBox Staging is highly configurable, well-optimized and fast enough to run
+      any old DOS game using modern hardware.
```

Comment:
well-optimized and fast enough are redundant, suggest:

DOSBox Staging is a highly configurable and fast enough to run any old DOS game using modern hardware.

or

DOSBox Staging is a highly configurable and sufficiently-optimized to run any old DOS game using modern hardware.

Reply:
Personally speaking, I'd go with the second example.